### ISSUE TITLE
lost-pixel.com resources return 404s

### ISSUE THREAD

**Issue by Maxim-Mazurok** [lost-pixel/lost-pixel#394]
### Bug description

https://lost-pixel.com/ lost styles and scripts, they return 404s:
![image](https://github.com/user-attachments/assets/150ca85b-8cb3-4e9f-a5c6-4ac9bcd4805d)

### How to reproduce

Open https://lost-pixel.com/

**Comment by d-ivashchuk** [comment]
hey @Maxim-Mazurok! what browser you were using or maybe there was anything specific like an incognito mode or anything that would help to narrow down the issue. can't reproduce it right now but also don't see any outages in our systems.

**Comment by Maxim-Mazurok** [comment]
@d-ivashchuk I was using latest Chrome on windows. Just tried latest Chrome on Android, on my wifi and on data, same result. I'm from Australia, maybe it's some geo issue?
![Screenshot_2024-08-04-11-32-23-952_com.android.chrome.jpg](https://github.com/user-attachments/assets/2b35740f-69c6-44a6-9b98-2b1683e82866)

**Comment by d-ivashchuk** [comment]
by any chance you have the network throttling enabled on your browsers? this is super weird as i can't reproduce it at all 😀

do you mind looking at network tab as the website loads please 🙏🏼 what happens there, anything sketchy?

**Comment by Maxim-Mazurok** [comment]
No fancy network setup, no adblock even, tried in incognito mode, both on PC and Phone.

There are examples of resources that return 404s:
- https://lost-pixel.com/_next/static/css/8756c315b1e6ea2e.css
- https://lost-pixel.com/_next/static/chunks/main-app-daa49b7b63cdc2f9.js

Note that WWW works fine: https://www.lost-pixel.com/
The non-WWW is the one I'm having issue with. 
You might have cached redirect, which is no longer in place, maybe?

Tho I tried to get screenshot from pagespeed insights for non-WWW and it looks fine... https://pagespeed.web.dev/analysis/https-lost-pixel-com/zo9m6vfvmg?use_original_url=true&form_factor=desktop
No idea mate, it must be GEO dependant then...

![image](https://github.com/user-attachments/assets/674ad291-2cc7-4b98-8b2e-093335c2f530)

**Comment by d-ivashchuk** [comment]
super weird, will give a more detailed look but not being able to reproduce it makes it hard to fix😀 will redeploy the landing page as well, maybe gets fixed

**Comment by Maxim-Mazurok** [comment]
Maybe try to use an Australian VPN?
I just checked on another laptop, the same thing.
Also I wonder where is it hosted, Vercel?

**Comment by Maxim-Mazurok** [comment]
I used free TunnelBear app on my android, it loads fine from US, Indonesia and Philippines, but not from Australian VPN. So there you go, can try that to reproduce.

**Comment by d-ivashchuk** [comment]
no, digital ocean for this one. i think on vercel might actually work with no troubles!

**Comment by jasikpark** [comment]
<img width="1154" alt="image" src="https://github.com/user-attachments/assets/7053eac0-56d8-4599-a759-d2826592962a">
seeing this today in Safari as well
<img width="636" alt="image" src="https://github.com/user-attachments/assets/1fe1c4c7-3bc1-4df2-9b2b-f42fc284f485">
most of the js isn't loading it seems? might be content-blocking issues, idk
